packages/vue-form-validator/package.json

```diff
@@ -25,25 +26,25 @@
     "typewatch": "vue-tsc --noEmit --watch",
     "lint": "yarn lint:js",
     "lint:fix": "yarn lint:js:fix",
-    "lint:js": "eslint --ext .js,.vue,.ts --ignore-path .gitignore .",
+    "lint:js": "eslint --ext .js,.vue,.ts .",
```

Comment:
is it needed to have `--ext` specified and having `files` configured in estling.config?